packages/kexec-tools/kexec-tools.spec

```
@@ -0,0 +1,32 @@
+Name: %{_cross_os}kexec-tools
+Version: 2.0.21
+Release: 1%{?dist}
+Summary: Linux tool to load kernels from the running system
+License: GPL-2.0
+URL: https://www.kernel.org/doc/html/latest/admin-guide/kdump/kdump.html
+Source0: https://kernel.org/pub/linux/utils/kernel/kexec/kexec-tools-%{version}.tar.xz
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}ncurses-devel
+
+%description
+%{summary}.
+
+%prep
+%setup -n kexec-tools-%{version}
+
+%build
+%cross_configure
+rm -f kexec-tools.spec.in
+make
```

Does this not build correctly with multiple jobs?

```suggestion
%make_build
```